Lib/test/test_tools/test_makefile.py

```diff
@@ -52,7 +52,10 @@ def test_makefile_test_folders(self):
             if not dirs and not files:
                 continue
             # Skip dirs with hidden-only files:
-            if files and all(filename.startswith('.') for filename in files):
+            if files and all(
+                filename.startswith('.') or filename in {'__pycache__'}
```

Comment:
Maybe a direct comparison is better here? (unless we're planning to ignore more filenames at some point)
```suggestion
                filename.startswith('.') or filename == '__pycache__'
```

Reply:
I don't think we are really planning any extras :)
So, yeah!